apps/fluent-tester/src/FluentTester/TestComponents/Shadow/ShadowButtonTestSection.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import { Platform, View } from 'react-native';
+import { FAB } from '@fluentui/react-native';
+import { commonTestStyles } from '../Common/styles';
+import { svgProps } from '../Common/iconExamples';
+
+export const ShadowButtonTestSection: React.FunctionComponent = () => {
+  if (Platform.OS === 'ios') {
+    return (
+      <View style={{ padding: 24, margin: 16 }}>
+        <FAB style={commonTestStyles.vmargin} icon={{ svgSource: svgProps, width: 20, height: 20 }}>
```

Comment:
Nit: move `{ svgSource: svgProps, width: 20, height: 20 }` to it's own variable / line so it's not recreated everytime this code runs, causing unnecessary re-renders